Kudu.Contracts/Jobs/JobSettingsKeys.cs

```diff
@@ -6,5 +6,6 @@ public static class JobSettingsKeys
         public const string StoppingWaitTime = "stopping_wait_time";
         public const string IsInPlace = "is_in_place";
         public const string Schedule = "schedule";
+        public const string IsWebHook = "is_webhook";
```

Comment:
I don't really like using the term 'passthrough' for this setting, because the WebJobs author doesn't see it as such.

Let's step back: how expensive is it to find an available port? If it's cheap enough, we should just do away with this setting and always do it. Are there any blatant drawbacks?